api/envoy/extensions/upstreams/http/v3/http_protocol_options.proto

```diff
@@ -150,7 +150,11 @@ message HttpProtocolOptions {
     AutoHttpConfig auto_config = 5;
   }
 
-  // [#not-implemented-hide:]
+  // .. warning::
```

Comment:
FWIW I don't like unhiding this before it's on, but now that I'm registering the codec filter, all our cross-checks require the field with extension-category be visible :-(

Reply:
As long as we document as done here and it's default disable I'm OK, @phlax for the challenge here.